charts/airbyte/values.yaml

```diff
@@ -90,21 +89,21 @@ webapp:
   ## ref: http://kubernetes.io/docs/user-guide/ingress/
   ## @param webapp.ingress.enabled Set to true to enable ingress record generation
   ## @param webapp.ingress.className Specifies ingressClassName for clusters >= 1.18+
-  ## @param webapp.ingress.hosts Ingress Hosts configuration
   ## @param webapp.ingress.annotations [object] Ingress annotations done as key:value pairs
   ## @param webapp.ingress.hosts The list of hostnames to be covered with this ingress record.
   ## @param webapp.ingress.tls [array] Custom ingress TLS configuration
   ingress:
     enabled: false
     className: ""
     annotations: {}
-      # kubernetes.io/ingress.class: nginx
-      # kubernetes.io/tls-acme: "true"
-    hosts:
-    - host: chart-example.local
-      paths:
-      - path: /
-        pathType: ImplementationSpecific
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+    hosts: []
```

Comment:
why is this commented out?

Reply:
Because it is only example and it breaks the `readme-generator` (it expect to have `hosts[0].pathType` dedicated comment)

Reply:
got it!